pkg/api/v1/persistentvolume/util_test.go

```diff
@@ -248,6 +247,9 @@ func collectSecretPaths(t *testing.T, path *field.Path, name string, tp reflect.
 	case reflect.Ptr:
 		secretPaths.Insert(collectSecretPaths(t, path, name, tp.Elem()).List()...)
 	case reflect.Struct:
+		if name == "ObjectMeta" {
```

Comment:
Add: `// objectMeta should not have any field with the word "secret" in it; it contains cycles so it's easiest to just skip it.`